server/src/main/resources/rules/rules.js

```diff
@@ -1400,6 +1416,38 @@ var Entitlement = {
         return this.build_func("do_pre_physical_only")();
     },
 
+    // pre_virt_only already covers is guest
+    do_pre_unmapped_guests_only: function(context, result) {
+        var caller = context.caller;
+        var consumer = context.consumer;
+        var unmapped_guest_pool = Utils.equalsIgnoreCase('true', context.getAttribute(context.pool, UNMAPPED_GUESTS_ONLY));
+
+        if (unmapped_guest_pool) {
+            if (context.hostConsumer){
+                if (BEST_POOLS_CALLER == caller ||
+                    BIND_CALLER == caller) {
+                    result.addError("virt.guest.cannot.use.unmapped.guest.pool.has.host");
+                }
+                else {
+                    result.addWarning("virt.guest.cannot.use.unmapped.guest.pool.has.host");
+                }
+            }
+            if (!context.newborn){
```

Comment:
Suspect this is unsafe use of context.newborn, it won't be set when these rules hit an older server via the manifest import. Just need to guard against it, unless we can come up with a way to calculate it in JS instead of always passing it in.